mixer/adapter/cloudwatch/config/config.proto

```diff
@@ -77,4 +81,19 @@ message Params {
       // [CloudWatch docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_MetricDatum.html)
       Unit unit = 3;
     }
+
+    // The name of the log group in cloudwatchlogs.
+    string log_group_name = 4;
```

Comment:
Do we need a note here (and below) saying "Required." ?

This relates to my other question about the ability to use one of metrics or logs without the other.

Reply:
This would be based on the discussion on previous comment.